src/promptflow/tests/test_configs/flows/chat-with-assistant-no-file/add_message_and_run.py

```diff
@@ -1,30 +1,32 @@
 import asyncio
 import json
+from typing import Union
 
-from openai import AsyncOpenAI
+from openai import AsyncAzureOpenAI, AsyncOpenAI
 from openai.types.beta.threads import MessageContentImageFile, MessageContentText
 
 from promptflow import tool, trace
-from promptflow.connections import OpenAIConnection
+from promptflow.connections import OpenAIConnection, _Connection, AzureOpenAIConnection
 from promptflow.contracts.multimedia import Image
 from promptflow.contracts.types import AssistantDefinition
 from promptflow.exceptions import SystemErrorException
 from promptflow.executor._assistant_tool_invoker import AssistantToolInvoker
+from get_assistant_client import get_assistant_client
 
 URL_PREFIX = "https://platform.openai.com/files/"
```

Comment:
When using AOAI, what is the output file url?

Reply:
they seems not provide way to peek into exact storage and  public url to access the file. For now of aoai, we only fill the file_id as url. Opened a ticket to follow up with aoai assistant support: https://ms.portal.azure.com/#view/Microsoft_Azure_Support/SupportRequestDetails.ReactView/id/%2Fsubscriptions%2F96aede12-2f73-41cb-b983-6d11a904839b%2Fproviders%2FMicrosoft.Support%2FsupportTickets%2F7d7bdc73-32f057d5-6b04497c-d47b-4298-9cd6-3c0cbff5707d